### PR TITLE
Mark resin.logs.clear() & resin.logs.historySinceClear() as deprecated

### PR DESCRIPTION
**Pull request by flesler**
Change-Type: patch
Connects-To: #527 

In preparation for #525 for v9
---- Autogenerated Waffleboard Connection: Connects to #527

**Question:** Should I regenerate the docs myself or will it be done automatically on release?

### CODE REVIEW

**Comment by LucianBuzzo** [comment]
@flesler Can you rebuild the docs please?

**Comment by flesler** [comment]
@LucianBuzzo I had asked above if I should or not. Now that I was able to fix the build locally I just did. All good now?

File: lib/logs.coffee [hunk 1]
```diff
@@ -193,6 +194,7 @@ getLogs = (deps, opts) ->
 	# @function
 	# @public
 	# @memberof resin.logs
+	# @deprecated use resin.logs.history() instead, since .clear() is deprecated
```
Comment (thgreasi) [inline lib/logs.coffee]: How about
> this method will be removed in the next major release, please use resin.logs.history() instead

File: lib/logs.coffee [hunk 2]
```diff
@@ -67,6 +67,7 @@ getLogs = (deps, opts) ->
 	# @summary Event fired when the logs have been cleared
 	# @event clear
 	# @memberof resin.logs.LogSubscription
+	# @deprecated since .clear() is deprecated, so is this event
```
Comment (thgreasi) [inline lib/logs.coffee]: How about simply
> this event will be removed in the next major release

File: lib/logs.coffee [hunk 3]
```diff
@@ -244,6 +246,7 @@ getLogs = (deps, opts) ->
 	# @function
 	# @public
 	# @memberof resin.logs
+	# @deprecated .clear() will be removed in the next major release
```
Comment (thgreasi) [inline lib/logs.coffee]: Ultra-NIT: I would use `this method` to be consistent with the rest deprecation warning of the SDK

File: lib/logs.coffee (hunk 1 above)
Comment (flesler) [reply lib/logs.coffee]: Changed

**Comment by flesler** [comment]
@thgreasi deleted the test in another commit

**Pull request merged**